pom.xml

```diff
@@ -267,7 +267,7 @@
         <storm.kafka.artifact.id>kafka_2.10</storm.kafka.artifact.id>
 
         <!-- kafka version used by new storm-kafka-client spout code -->
-        <storm.kafka.client.version>0.10.0.0</storm.kafka.client.version>
+        <storm.kafka.client.version>0.10.1.1</storm.kafka.client.version>
```

Comment:
Lets keep the version to 0.10.0.1 here. Its better to support lowest version as much as possible. As of now old clients can connect to new brokers but not other way round.